koin-projects/koin-core/src/main/kotlin/org/koin/core/parameter/DefinitionParameters.kt

```diff
@@ -28,7 +28,7 @@ import kotlin.reflect.KClass
  * @author - Arnaud GIULIANI
  */
 @Suppress("UNCHECKED_CAST")
-open class DefinitionParameters(vararg val values: Any?) {
+open class DefinitionParameters(val values: MutableList<Any> = arrayListOf()) {
```

Comment:
Backward compatibility breaking change

Reply:
no problem here, parametersOf is still the same